### ISSUE TITLE
Is fine tuning on the pre-trained model provided a viable solution?

### ISSUE THREAD

**Issue by TommasoBendinelli** [j96w/DenseFusion#133]
Hello,
I would like to do pose estimation of custom workpieces. I was thinking, as we do for mask-cnn, if I could just download the pretrained model and then fine tune with my dataset, instead of training from scratch.  Is it a viable approach?

**Comment by j96w** [comment]
Hi, yes, for different segmentation results finetuning, it is viable.

**Comment by akeaveny** [comment]
Hi @j96w, just to confirm there isn't a similar method to finetune results on a target domain with pose estimation?
